src/evops/utils/IoUOverlap.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2022, Skolkovo Institute of Science and Technology (Skoltech)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any
+from nptyping import NDArray
+
+import numpy as np
+
+from evops.metrics.IoUBenchmark import __iou
+from evops.metrics.constants import IOU_THRESHOLD
+
+
+def __iou_overlap(
+    pred_labels: NDArray[Any, np.int32],
+    gt_labels: NDArray[Any, np.int32],
+) -> (np.int32, np.int32, np.int32):
+    """
+    :param pred_labels: labels of points corresponding to segmented planes
+    :param gt_labels: indices of points corresponding to ground truth planes
+    :return: (true positive, false positive, false negative) received using pred_indices and gt_indices
+    """
+    true_positive = 0
```

Comment:
We can calculate only true positive

Reply:
Doesn't the rule "e.g. 5 detections (TP) of a single object is counted as 1 correct detection and 4 false detections" impose additional conditions?